### PR TITLE
Docs/docker network

### PR DESCRIPTION
**Pull request by apenzk**
# Description of change

Moving the doc file to the correct location

### CODE REVIEW

File: docs/tooling/docker_private_network.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+```
+./get_approval_csv.sh
+```
+Example output:
+```
+Triggering approval analysis on peer_master and 20 replicas...
+Triggering approval analysis on peer_master and 20 replicas... DONE
+Copying csv files from peer_master and 20 replicas...
+Copying csv files from peer_master and 20 replicas... DONE
+Copied files are located at /home/{user}/repo/goshimmer/tools/docker-network/csv
```
Comment (jonastheis) [inline docs/tooling/docker_private_network.md:54]: This should be given as a relative path and changed to be a relative path if that's an absolute path in the tool itself.

**Review by jonastheis — COMMENTED** [review]
The tool additionally supports the option to be run with Grafana enabled `Call with ./run replicas [grafana=0|1]`. Optimally, this should be documented and shown in the figure as well as how to access and use Grafana.

File: docs/tooling/docker_private_network.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+We provide a tool at `tools/docker-network` with which a local test network can be set up locally with docker. 
+ 
+![Docker network](../../images/docker-network.png)
+
+
+## How to use the tool
+
+In the docker network run for example
+```
+./run.sh 5
```
Comment (jkrvivian) [inline docs/tooling/docker_private_network.md:12]: `./run.sh 5 1`, the second argument is for grafana
* default or `0`: disable
* `1`: enable

File: docs/tooling/docker_private_network.md (hunk 1 above)
Comment (apenzk) [reply docs/tooling/docker_private_network.md]: Thanks Jonas, I have applied the suggested change

File: docs/tooling/docker_private_network.md (hunk 2 above)
Comment (apenzk) [reply docs/tooling/docker_private_network.md]: done with reference to the doc where grafana is described in more details

**Review by jkrvivian — APPROVED** [review]

**Pull request merged**